api/desecapi/models.py

```diff
@@ -227,3 +230,68 @@ def save(self, *args, **kwargs):
 
     class Meta:
         ordering = ('created',)
+
+
+class RRset(models.Model):
+    # TODO Do these two fields really make sense? Meaning is limited when deleting + recreating an RRset
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(null=True)
+    domain = models.ForeignKey(Domain, on_delete=models.CASCADE, related_name='rrsets')
+    subname = models.CharField(max_length=178, blank=True)
+    type = models.CharField(max_length=10)
+    records = models.CharField(max_length=64000, blank=True)
```

Comment:
Our interface provides access to RRsets, not to individual resource records. Also internally in our model, individual resource records are not accessible. Therefore, each (JSON representation of an) RRset is atomic. The DBMS or even our application are not concerned with accessing sub-parts of the `records` field. We receive it as a whole from the user, and pass it on to pdns.

(On the way, we check whether the value is well-formed JSON. However, it is common to validate atomic values, such as domain names, phone numbers, ...)

Thus, in the context of what we're doing, we are storing one value here (i.e., the list of record values).